recipe/build.sh

```diff
@@ -4,17 +4,62 @@ set -euxo pipefail
 
 if [[ "${target_platform}" == osx-* ]]; then
   export LDFLAGS="${LDFLAGS} -lz -framework CoreFoundation -Xlinker -undefined -Xlinker dynamic_lookup"
+  # Remove stdlib=libc++; this is the default and errors on C sources.
+  export CXXFLAGS=${CXXFLAGS//-stdlib=libc++}
```

Comment:
This PR is being built only for linux-64 but this is an osx change - just wondering why this was found and added during this build process and why the previous osx builds were ok?

Reply:
It was mostly to be more aligned with the conda forge recipe. But it can be removed.